src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -141,9 +144,9 @@ interface IProps {
 }
 
 @replaceableComponent("views.rooms.SendMessageComposer")
-export default class SendMessageComposer extends React.Component<IProps> {
-    static contextType = MatrixClientContext;
-    context!: React.ContextType<typeof MatrixClientContext>;
+class SendMessageComposer extends React.Component<ISendMessageComposerProps> {
+    static contextType = RoomContext;
+    context!: React.ContextType<typeof RoomContext>;
```

Comment:
I don't think I've come across that `!:` syntax before, what does it do 🤔 
Tried Googling a few things, and that didn't bring up a lot of results

Reply:
Declares that the value will always be defined, like doing `this.value!`
Probably unnecessary though. Removed!